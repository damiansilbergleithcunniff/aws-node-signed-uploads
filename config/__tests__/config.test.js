const devConfig = require('../config.dev.json');
const testConfig = require('../config.test.json');
const stageConfig = require('../config.stage.json');
const prodConfig = require('../config.prod.json');

describe('Config Files', () => {
  test('dev config to be correct', () => {
    expect(devConfig).toMatchSnapshot();
  });

  test('test config to be correct', () => {
    expect(testConfig).toMatchSnapshot();
  });

  test('stage config to be correct', () => {
    expect(stageConfig).toMatchSnapshot();
  });

  test('prod config to be correct', () => {
    expect(prodConfig).toMatchSnapshot();
  });
});
