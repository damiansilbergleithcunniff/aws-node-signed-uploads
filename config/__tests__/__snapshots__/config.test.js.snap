// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config Files dev config to be correct 1`] = `
Object {
  "region": "us-east-1",
  "someval": "hello nurse",
  "stage": "dev",
}
`;

exports[`Config Files prod config to be correct 1`] = `Object {}`;

exports[`Config Files stage config to be correct 1`] = `Object {}`;

exports[`Config Files test config to be correct 1`] = `
Object {
  "region": "us-east-1",
  "someval": "hello test",
  "stage": "dev",
}
`;
